client/main.lua

```diff
@@ -591,7 +596,7 @@ end
 -- Events
 
 RegisterNetEvent('hospital:client:ambulanceAlert', function(coords, text)
-    local street1, street2 = GetStreetNameAtCoord(coords.x, coords.y, coords.z)
```

Comment:
No

Reply:
My bad, I copied his code and he hasn't updated, fixed it though